File: official/docs/node/v6/carrier-accounts/delete.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const carrierAccount = await client.CarrierAccount.retrieve('ca_...');

  await client.CarrierAccount.delete(carrierAccount.id);
})();

File: official/docs/golang/current/webhooks/delete.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func delete() {
	client := easypost.New("EASYPOST_API_KEY")

	err := client.DeleteWebhook("hook_...")

	fmt.Println(err)
}
